### PR TITLE
[Feature] Provide game scores from PCGamingWiki and other useful information

### PR DESCRIPTION
**Pull request by Nocccer**
Added a function to get extra game infomration from PCGamingWiki.
Currently game scores, direct 3D version, howlongtobeat ID and steam ID are fetched.

Game scores are added to gamepage:
image
image
- Red if under 33. Had no game under this score.

Make use of Steam id for protondb:

https://user-images.githubusercontent.com/61798668/210016244-c660836d-d93c-47b6-85ea-97a2224ecea6.mp4

- To get the data the PCGamingWiki API is used. https://www.pcgamingwiki.com/wiki/PCGamingWiki:API
- PCGamingWiki provide searching via gog id. Gog games may are found better than epic games.

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [x] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by flavioislima** [comment]
The idea is cool but I do not think the design is 😅 
We would need to think about how to handle that in a different way.

File: public/locales/ar/gamepage.json [hunk 1]
```diff
@@ -93,6 +93,12 @@
         "moving": "قيد النقل",
         "repairing": "قيد الإصلاح"
     },
+    "gamescore": {
+        "header": "Game Scores",
+        "igdb": "IGDB",
+        "metacritic": "MetaCritic",
```
Comment (flavioislima) [inline public/locales/ar/gamepage.json:99]: Maybe these texts does not need translations, just like Discord, Patreon, for instance.

**Comment by Nocccer** [comment]
> The idea is cool but I do not think the design is sweat_smile We would need to think about how to handle that in a different way.

I added some colors. Still feel free to adapt the design. I am not a good designer :D.
I also make use of the steam id for protondb. Works 95% or more. Tested like 15 games.

**Comment by flavioislima** [comment]
@Nocccer what do you think of that?
image

Only a temporary approach since we will change the whole game page now. But for the current version might be a bit better I guess.

**Comment by Nocccer** [comment]
> @Nocccer what do you think of that? image
> 
> Only a temporary approach since we will change the whole game page now. But for the current version might be a bit better I guess.
> 
> Let me know if I can push this change

Looks good. You can push.

**Comment by flavioislima** [comment]
Also, do you think would be too much work to go to the website in question when clicking on the circle?

**Comment by Nocccer** [comment]
> Also, do you think would be too much work to go to the website in question when clicking on the circle?

Which website you mean? PCgamingWiki does not provide the link to the gamescore pages.

**Comment by flavioislima** [comment]
> > Also, do you think would be too much work to go to the website in question when clicking on the circle?
> 
> Which website you mean? PCgamingWiki does not provide the link to the gamescore pages.

Ah I see, no problem then. Lets keep as it is 👍🏽

File: public/locales/fr/translation.json [hunk 2]
```diff
@@ -255,7 +255,7 @@
...
             "part2": "Dans le cas où vous changez le dossier de prefix de Wine ou Proton, vous aurez besoin de revérifier ce chemin car Proton utilise un prefix différent (/pfx) et un username (steamuser). Vous pouvez simplement effacer le chemin actuel quitter la page des options pour que Heroic essaie de deviner une fois de plus le dossier avec le bon prefix.",
-            "part3": "Synchronisation manuelle : Choisissez « Télécharger » pour télécharger les sauvegardes des jeux enregistrées dans le Cloud et « Téléverser » pour transférer les fichiers locaux vers le Cloud. « Forcer le téléchargement » et « Forcer le téléversement » ignoreront la version locale ou celle du Cloud.",
+            "part3": "Synchronisation manuelle\u00a0: Choisissez « Télécharger » pour télécharger les sauvegardes des jeux enregistrées dans le Cloud et « Téléverser » pour transférer les fichiers locaux vers le Cloud. « Forcer le téléchargement » et « Forcer le téléversement » ignoreront la version locale ou celle du Cloud.",
```
Comment (flavioislima) [inline public/locales/fr/translation.json:258]: remove those weird characters, might be more on other files.

**Comment by Nocccer** [comment]
> > > Also, do you think would be too much work to go to the website in question when clicking on the circle?
> > 
> > Which website you mean? PCgamingWiki does not provide the link to the gamescore pages.
> 
> Ah I see, no problem then. Lets keep as it is 👍🏽 

Let me check tomorrow. Maybe they do.

**Review by flavioislima — APPROVED** [review]
LGTM, just fix those small issues with the translations files

**Comment by Nocccer** [comment]
> LGTM, just fix those small issues with the translations files

Can you check back? I added the urls. Maybe some css is needed to indicate it is clickable. I think something that can be done in the design PR ?

**Comment by flavioislima** [comment]
> > LGTM, just fix those small issues with the translations files
> 
> Can you check back? I added the urls. Maybe some css is needed to indicate it is clickable. I think something that can be done in the design PR ?

Sure, I will check when I am on the computer. 👍

**Comment by flavioislima** [comment]
It is working fine now.
Just added some small UI fixes because it was showing the GameScore title even if no score was coming from the API.
And also when the HLTB data as missing, it was breaking the layout.

**Pull request merged**